cgmanifest.json

```diff
@@ -6385,8 +6385,8 @@
         "type": "other",
         "other": {
           "name": "rust",
-          "version": "1.50.0",
-          "downloadUrl": "https://static.rust-lang.org/dist/rustc-1.50.0-src.tar.xz"
+          "version": "1.47.0",
```

Comment:
Are we rolling back rust, or supporting both versions?

Reply:
No, we are upgrading from 1.39.0 to 1.47.0.
I added a comment to the description explaining the use of 1.47.0 instead of 1.50.0